doc/tasks/symfony_console.md

```diff
@@ -0,0 +1,96 @@
+# Symfony Console
+
+Run a symfony console command.
+
+## Composer
+
+Requires the Symfony Console component: [Console Component](https://symfony.com/components/Console)
+
+```bash
+composer require symfony/console
```

Comment:
This is only valid if you are using flex:
https://github.com/symfony/recipes/tree/main/symfony/console/5.3

Maybe easier to tell that they should be operating in a symfony application?
The suggest in composer.json might be removed as well maybe? Since It's rather : have the framework installed than have a specific dependency.